webcompat/static/config_data/labels.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "url": "https://api.github.com/repos/webcompat/web-bugs/labels/status-duplicate",
+    "name": "status-duplicate",
+    "color": "cccccc"
```

Comment:
I think we probably don't want to be in the business of serving hand-written JSON. 

What about building this file programatically when the app starts from something like labels-config.py. We could hash the file to see if it changed that way we don't have to re-build the file unless we know there are new labels.

All we need in labels-config.py is a label name and a color.